Files/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -656,6 +657,37 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                     IsPrimary = true,
                 },
                 new ContextMenuFlyoutItemViewModel()
+                {
+                    Text = "BaseLayoutItemContextFlyoutExtractionOptions".GetLocalized(),
+                    Glyph = "\xF11A",
+                    ShowItem = selectedItems.Count == 1 && selectedItems.First().FileExtension == ".zip",
+                    GlyphFontFamilyName = "CustomGlyph",
+                    Items = new List<ContextMenuFlyoutItemViewModel>()
+                    {
+                        new ContextMenuFlyoutItemViewModel()
+                        {
+                            Text = "BaseLayoutItemContextFlyoutExtractFilesOption".GetLocalized(),
```

Comment:
Now that we have this menu item, should we hide the shell option that we currently display?  Fyi @gave92

Reply:
Done!

Reply:
Thanks!